plugins/woocommerce/src/Admin/Features/ProductBlockEditor/BlockRegistry.php

```diff
@@ -20,9 +20,13 @@ class BlockRegistry {
 	 * Array of all available product blocks.
 	 */
 	const PRODUCT_BLOCKS = [
-		'woocommerce/product-name',
-		'woocommerce/product-pricing',
+		'woocommerce/product-collapsible',
```

Comment:
So this clearly isn't an exhaustive list of our product blocks; do you know if we should be updating this as we add blocks? It would be nice if we could automate that in some way since it will be easy to forget.

Reply:
Yep I think this is something we always forget. I just included only the block I touched in this PR. But I can add the resting ones.